api/globalSearch/index.spec.ts

```diff
@@ -9,26 +8,142 @@ import { mockReq, mockRes } from 'sinon-express-mock';
-    { id: 'IA', name: 'Immigration & Asylum', description: null, caseTypes: null },
-    { id: 'PUBLICLAW', name: 'Public Law', description: null, caseTypes: null },
-    { id: 'DIVORCE', name: 'Family Divorce', description: null, caseTypes: null },
-    { id: 'PRIVATELAW', name: 'PRIVATE LAW', description: null, caseTypes: null },
-    { id: 'EMPLOYMENT', name: 'EMPLOYMENT', description: null, caseTypes: null }
+  const fefDataHMCTS: RefDataHMCTSService[] = [
+    {
+      jurisdiction: 'Immigration and Asylum Chamber',
+      service_id: 39,
+      org_unit: 'HMCTS',
+      business_area: 'Civil, Family and Tribunals',
+      sub_business_area: 'Tribunals',
+      service_description: 'Immigration and Asylum Appeals',
+      service_code: 'BFA1',
+      service_short_description: 'Immigration and Asylum Appeals',
+      ccd_service_name: 'IA',
+      last_update: '2021-02-10T11:53:58.808063',
+      ccd_case_types: ['Asylum', 'Bail']
+    },
+    {
+      jurisdiction: 'Civil',
+      service_id: 6,
+      org_unit: 'HMCTS',
+      business_area: 'Civil, Family and Tribunals',
+      sub_business_area: 'Civil and Family',
+      service_description: 'Specified Money Claims',
+      service_code: 'AAA6',
+      service_short_description: 'Specified Money Claims',
+      ccd_service_name: 'Civil',
+      last_update: '2021-02-10T11:53:58.808063',
+      ccd_case_types: [
+        'MoneyClaimCase',
+        'CMC_ExceptionRecord',
+        'CIVIL',
+        'GENERALAPPLICATION'
+      ]
+    },
+    {
+      jurisdiction: 'Civil',
+      service_id: 7,
+      org_unit: 'HMCTS',
+      business_area: 'Civil, Family and Tribunals',
+      sub_business_area: 'Civil and Family',
+      service_description: 'Damages',
+      service_code: 'AAA7',
+      service_short_description: 'Damages',
+      ccd_service_name: 'CIVIL',
+      last_update: '2021-02-10T11:53:58.808063',
+      ccd_case_types: ['UNSPECIFIED_CLAIMS', 'CIVIL', 'GENERALAPPLICATION']
+    },
+    {
+      jurisdiction: 'Family',
+      service_id: 12,
+      org_unit: 'HMCTS',
+      business_area: 'Civil, Family and Tribunals',
+      sub_business_area: 'Civil and Family',
+      service_description: 'Family Private Law',
+      service_code: 'ABA5',
+      service_short_description: 'Family Private Law',
+      ccd_service_name: 'PRIVATELAW',
+      last_update: '2021-02-10T11:53:58.808063',
+      ccd_case_types: ['PRLAPPS']
+    },
+    {
+      jurisdiction: 'Family',
+      service_id: 10,
+      org_unit: 'HMCTS',
+      business_area: 'Civil, Family and Tribunals',
+      sub_business_area: 'Civil and Family',
+      service_description: 'Family Public Law',
+      service_code: 'ABA3',
+      service_short_description: 'Family Public Law',
+      ccd_service_name: 'PUBLICLAW',
+      last_update: '2021-02-10T11:53:58.808063',
+      ccd_case_types: ['CARE_SUPERVISION_EPO']
+    },
+    {
+      jurisdiction: 'Employment Tribunerals',
```

Comment:
Not important, but typo in Employment Tribunals

Reply:
Changed